assets/js/blocks/reviews-by-product/block.js

```diff
@@ -173,12 +220,32 @@ class ReviewsByProduct extends Component {
 					/>
 				</BlockControls>
 				{ this.getInspectorControls() }
-				{ editMode ? (
+				{ ! product || editMode ? (
 					this.renderEditMode()
 				) : (
-					<Disabled>
-						<ServerSideRender block={ name } attributes={ attributes } className="wc-block-reviews-by-product" />
-					</Disabled>
+					<Fragment>
+						{ !! product.rating_count ? (
```

Comment:
I guess there is no need for the double negation here, it will try to evaluate if `product.rating_count` is truthy or falsy anyway:
```suggestion
						{ product.rating_count ? (
```
or checking that is higher than 0 would also work:
```suggestion
						{ product.rating_count > 0 ? (
```